vendor/engines/split_accounts/spec/models/split_account_spec.rb

```diff
@@ -58,8 +61,23 @@
     context "when splits total 100 and multiple splits have extra_penny" do
       let(:split_account) do
         build(:split_account, without_splits: true).tap do |split_account|
-          split_account.splits << build(:split, percent: 50, extra_penny: true, parent_split_account: split_account)
-          split_account.splits << build(:split, percent: 50, extra_penny: true, parent_split_account: split_account)
+          split_account.splits << build(:split, percent: 50, extra_penny: true, parent_split_account: split_account, subaccount: subaccount_1)
+          split_account.splits << build(:split, percent: 50, extra_penny: true, parent_split_account: split_account, subaccount: subaccount_2)
+        end
+      end
+
+      it "is invalid" do
+        expect(split_account).not_to be_valid
+        expect(split_account.errors).to include(:splits)
```

Comment:
I feel like now that we're adding more validations, we should probably check that it's invalid for the right reason.